charts/celestia-node/values.yaml

```diff
@@ -74,6 +74,12 @@ diagnosticMode:
 ## %%MAIN_CONTAINER/POD_DESCRIPTION%%
 ##
 node:
+  ## Select from: bridge, full, light
+  ## Valid values:
+  ## - bridge
+  ## - full
+  ## - light:
+  nodeType: bridge
```

Comment:
This should also change the args here: https://github.com/celestiaorg/helm-charts/blob/ac54d08c18c96ad5c87796c34b8d9e008f1f2fa8/charts/celestia-node/values.yaml#L511

This way, we can start different node types without touching the `args` in the values file.